### PR TITLE
Create ELAN content types

### PR DESCRIPTION
**Pull request by lzehl**
ELAN content types:
+ vnd.elan.event (.pos)
+ vnd.elan.continuous-data (.eeg + .eeg.ent)

### CODE REVIEW

File: instances/data/contentTypes/elan.continuous-data.jsonld [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	"fileExtension": [
+		".eeg",
+    		".eeg.ent"
+	],
+	"name": "application/vnd.elan.continuous-data",
+	"relatedMediaType": null,
+	"synonym": [
+		"ELAN continuous data file"
+	]
+}
```
Comment (jagru20) [inline instances/data/contentTypes/elan.continuous-data.jsonld:16]: does this file extend or replace the existing elan contenttype?

**Review by jagru20 — CHANGES_REQUESTED** [review]
This PR should go into v4, right? We should change the default branch.

File: instances/data/contentTypes/elan.continuous-data.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/data/contentTypes/elan.continuous-data.jsonld:16]: replace.

Comment (lzehl) [reply instances/data/contentTypes/elan.continuous-data.jsonld:16]: well. one replaces the other is an additional elan content type

**Comment by lzehl** [comment]
> 

damn, it should go into v4 yes... yes we should really change the default yes. (I'll take care of it after this PR).

**Comment by lzehl** [comment]
@jagru20 I've changed the base. Something else I missed?

File: instances/data/contentTypes/elan.continuous-data.jsonld (hunk 1 above)
Comment (jagru20) [reply instances/data/contentTypes/elan.continuous-data.jsonld:16]: @lzehl  did you see this?

Comment (lzehl) [reply instances/data/contentTypes/elan.continuous-data.jsonld:16]: what do you mean?

Comment (jagru20) [reply instances/data/contentTypes/elan.continuous-data.jsonld:16]: oh it did not display your response, while i was writing that. next time I'll refresh the page before commenting :D

Comment (jagru20) [reply instances/data/contentTypes/elan.continuous-data.jsonld:16]: However, then the other file needs to be deleted

**Review by jagru20 — APPROVED** [review]

**Comment by lzehl** [comment]
@jagru20 thank you for taking care of deleting the existing one... I completely overlooked (and actually thought I modified the existing one...) Should I merge then?

**Comment by lzehl** [comment]
@jagru20 can we merge this one?

**Comment by jagru20** [comment]
sorry, I missed that last comment! yes we can merge it.

**Pull request merged**